src/views/Main/Head/Head.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+
+const Head = ({ settings = {} }) => {
```

Comment:
You may spread theme here ?

```js

const Head = ({ settings: {theme:{}={}, title} = {} }) => {
```

Reply:
He would probably write:
const Head = ({ settings: { theme: { style } ={}, title } = {} }) => {

Reply:
done

Reply:
done